src/cppkin.h

```diff
@@ -16,8 +16,7 @@
 
 
 #define INIT(params) \
-    core::Environment::Instance().Init(); \
```

Comment:
Remind me why you needed to move this to cpp?

Reply:
Environment contains a global scope external linkage member. If Core will be staticly linked to an App which will also use cppkin, you will have 2 Environments instances. Linux dynamic linker knows how to merge the shared one into the applicative one (done via the GOT section), in windows it needs to be done explictly, Init() function is exported/imported in windows explicitly, which solves the issue.